testrail/api.py

```diff
@@ -175,17 +175,21 @@ def _refresh(self, ts):
 
     @classmethod
     def flush_cache(cls):
-        """ Set all cache objects to refresh
+        """ Set all cache objects to refresh the next time they are accessed
         """
+        def clear_ts(cache):
+            if 'ts' in cache:
```

Comment:
Is there a reason that we need to check if the timestamp exists? I know this was in the previous code but I think just setting cache['ts'] to None always would be fine. Or am I forgetting something?